themes/react-data-grid.css

```diff
@@ -295,105 +295,232 @@ input.react-grid-CheckBox {
+ * MIT License: https://github.com/keystonejs/react-select
+*/
+.Select {
+  position: relative;
+}
+.Select-control {
+  position: relative;
+  overflow: hidden;
   background-color: white;
-  background-image: none;
   border: 1px solid #cccccc;
+  border-color: #d9d9d9 #cccccc #b3b3b3;
   border-radius: 4px;
-  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
-  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
-  -webkit-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
-  -o-transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
-  transition: border-color ease-in-out .15s, box-shadow ease-in-out .15s;
+  box-sizing: border-box;
+  color: #333333;
+  cursor: default;
+  outline: none;
+  padding: 8px 52px 8px 10px;
+  transition: all 200ms ease;
 }
-.react-autocomplete-Autocomplete__search:focus {
-  border-color: #a21618;
-  outline: 0;
-  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(162, 22, 24, 0.6);
-  box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(162, 22, 24, 0.6);
+.Select-control:hover {
+  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.06);
 }
-.react-autocomplete-Autocomplete__search::-moz-placeholder {
-  color: #777777;
-  opacity: 1;
+.is-searchable.is-open > .Select-control {
+  cursor: text;
 }
-.react-autocomplete-Autocomplete__search:-ms-input-placeholder {
-  color: #777777;
+.is-open > .Select-control {
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+  background: white;
+  border-color: #b3b3b3 #cccccc #d9d9d9;
 }
-.react-autocomplete-Autocomplete__search::-webkit-input-placeholder {
-  color: #777777;
+.is-open > .Select-control > .Select-arrow {
+  border-color: transparent transparent #999999;
+  border-width: 0 5px 5px;
 }
-.react-autocomplete-Autocomplete__search[disabled],
-.react-autocomplete-Autocomplete__search[readonly],
-fieldset[disabled] .react-autocomplete-Autocomplete__search {
-  cursor: not-allowed;
-  background-color: #eeeeee;
-  opacity: 1;
+.is-searchable.is-focused:not(.is-open) > .Select-control {
+  cursor: text;
 }
-textarea.react-autocomplete-Autocomplete__search {
-  height: auto;
+.is-focused:not(.is-open) > .Select-control {
+  border-color: #0088cc #0099e6 #0099e6;
+  box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1), 0 0 5px -1px rgba(0, 136, 204, 0.5);
 }
-.react-autocomplete-Autocomplete__results {
+.Select-placeholder {
+  color: #aaaaaa;
+  padding: 8px 52px 8px 10px;
   position: absolute;
-  top: 100%;
+  top: 0;
   left: 0;
-  z-index: 1000;
-  display: none;
-  float: left;
-  min-width: 160px;
-  padding: 5px 0;
-  margin: 2px 0 0;
-  list-style: none;
-  font-size: 14px;
-  text-align: left;
+  max-width: 100%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+.has-value > .Select-control > .Select-placeholder {
+  color: #333333;
+}
+.Select-input > input {
+  cursor: default;
```

Comment:
Should probably remove this from CSS file, and say that React-Select is a dependency that needs to be included for react-data-grid-with-addons